frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+import Location from './Location';
+import DateTime from './DateTime';
+import StatusToggle from './StatusToggle';
+import Dropdown from './Dropdown';
+import './CrossingListItem.css';
+
+class CrossingListItem extends React.Component {
+
+  render () {
+    var show = [];
+    switch(this.props.status) {
+      case 'Open':
```

Comment:
This really shouldn't be using a string. Definitely need to refactor when setting status from the status toggle.